include/taskolib/serialize_sequence.h

```diff
@@ -118,6 +118,16 @@ std::ostream& operator<<(std::ostream& stream, const Sequence& sequence);
  */
 void store_sequence(const std::filesystem::path& folder, const Sequence& sequence);
 
+
+/**
+ * Removes the Sequence by deleting the folder
+ *
+ *
+ * \param folder  in which the Sequence is stored
+ * \param sequence  the Sequence that should be removed
+ */
+void remove_sequence(const std::filesystem::path& seq_path);
```

Comment:
No reason. In Sequence Manager I want to have a delete_sequence with version control and I did not know where to put the untracked one. Function is not used, can also be deleted.

Reply:
Oh, I understand. There are many possible ways to go about this, I guess. I would tend towards not duplicating all the file managing (renaming, removing) functionality. The low-level functions for storing and loading a sequence to/from a folder should be accessible for users who want to do something out-of-the-ordinary, but otherwise I would just provide everything through the `SequenceManager`, with git integration. @MissPeppermintPatty, @wyrnat: Do you agree?

Reply:
Okay, then let's remove `remove_sequence()` here.